Add favicons to /images/exampes

caugner opened the pull request:
<!-- 🙌 Thanks for contributing! Adding details below will help us to merge your PR faster. -->

**Description:**

Adds favicon PNGs from content to `/images/examples`.

**Motivation:**

Allows us to replace several broken image refs.

**Additional details:**

<!-- 🔗 Link to documentation, bug trackers, source control, or other places providing more context -->

**Related issues and pull requests:**

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->

caugner committed:
Add favicons to /images/exampes

bsmth reviewed (approved):
ty

caugner committed:
Add favicon144.png

Source: https://web.archive.org/web/20171101120729/https://developer.mozilla.org/static/img/favicon114.png

caugner committed:
Add favicon57.png

Source: https://web.archive.org/web/20171101113241/https://developer.mozilla.org/static/img/favicon57.png

caugner commented:
@bsmth FYI I found two more favicons linked from translated-content.

bsmth reviewed (approved):
Additions look good, tnx

caugner committed:
Add web-docs-sprite.svg

Source: https://web.archive.org/web/20180227004606/https://developer.mozilla.org/static/img/web-docs-sprite.svg
